pkg/agent/manifest.go

```diff
@@ -32,7 +32,7 @@ func Manifest(namespace, image, pullPolicy, generation, checkInInterval string)
 		},
 	)
 
-	dep := basic.Deployment(namespace, DefaultName, image, pullPolicy, DefaultName)
+	dep := basic.Deployment(namespace, DefaultName, image, pullPolicy, DefaultName, false)
```

Comment:
We shouldn't need this once agent image support windows, correct?

Reply:
Once the `rancher/fleet` (manager) image does, then correct, we will not need this. However, since the manger image does not support Windows currently, we need to be able to create...

1. a manager deployment: needs to be scheduled on Linux nodes
2. an agent deployment: has a Windows manifest